<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AccessAnalysisWidget</class>
 <widget class="QWidget" name="AccessAnalysisWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>586</width>
    <height>501</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AccessAnalysisWidget</string>
  </property>
  <widget class="QLabel" name="oriLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>出发点</string>
   </property>
  </widget>
  <widget class="QLabel" name="dirPointLayer">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>目标点图层</string>
   </property>
  </widget>
  <widget class="QLabel" name="arcLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>道路表</string>
   </property>
  </widget>
  <widget class="QLabel" name="nodeLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>220</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>交叉路口表</string>
   </property>
  </widget>
  <widget class="QLabel" name="timeLimitLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>334</y>
     <width>72</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>时间范围</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="timeLimitTextLine">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>360</y>
     <width>501</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>440</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QComboBox" name="arcBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>180</y>
     <width>501</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>440</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLabel" name="unitLabel">
   <property name="geometry">
    <rect>
     <x>471</x>
     <y>390</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>单位：分钟</string>
   </property>
  </widget>
  <widget class="QComboBox" name="nodeBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>240</y>
     <width>501</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="dirPointLayerBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>501</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="oriBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>501</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="saveFieldTextLine">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>300</y>
     <width>501</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="saveFieldLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>275</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>结果储存字段</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
